geocity/apps/submissions/management/commands/update_payment_transactions_status.py

```diff
@@ -0,0 +1,56 @@
+import datetime
+
+from django.conf import settings
+from django.core.management import BaseCommand
+from django.utils import timezone
+from django.utils.translation import gettext
+
+from geocity.apps.submissions.payments.models import Transaction
+from geocity.apps.submissions.payments.postfinance.models import PostFinanceTransaction
+from geocity.apps.submissions.payments.services import get_payment_processor
+
+
+class Command(BaseCommand):
+    help = gettext(
+        "Update the status of transactions that are pending and not older than %s hours."
+        % settings.PAYMENT_PENDING_TRANSACTION_MAX_AGE_MINS
+    )
+
+    def handle(self, *args, **options):
+        self.stdout.write(
+            "Checking status of unpaid transations that are not older than %s minutes..."
+            % settings.PAYMENT_PENDING_TRANSACTION_MAX_AGE_MINS
+        )
+
+        nb_transactions_confirmed = 0
+        nb_transactions_failed = 0
+        # Get all unpaid transactions that are not older than the specified time
+        transactions_to_update = PostFinanceTransaction.objects.filter(
+            status=Transaction.STATUS_UNPAID,
+            authorization_timeout_on__gte=timezone.now()
```

Comment:
There is an older issue here, for a reason I don't get, the value of this attribute in DB is truncated to the day, the time is not stored (2024-06-12 --> 2024-06-12 00:00:00.000 +0200). So with the default value of 30 minutes, it will never work.

If we use the value as provided in `env.example`, it works because it's 24 hours and not 30 minutes.

But it's not clean, is there a way to store the timeout value with the hours?